src/vcpkg/commands.ci.cpp

```diff
@@ -276,6 +279,7 @@ namespace vcpkg::Commands::CI
         std::map<PackageSpec, std::vector<std::string>> features;
         Dependencies::ActionPlan plan;
         std::map<PackageSpec, std::string> abi_map;
+        int cascade_count = 0;
```

Comment:
Is there a reason that we want `cascade_count` to be an `int` and not, say, a `size_t`?

Reply:
It isn't an in memory thing and we already have a `strto` explicit specialization for int (but not for size_t).